<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="mainButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: grey</string>
   </property>
   <property name="text">
    <string>메인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reservationButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
</string>
   </property>
   <property name="text">
    <string>예약 관리</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="scheduleButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
</string>
   </property>
   <property name="text">
    <string>일정 관리</string>
   </property>
  </widget>
  <widget class="QPushButton" name="profileButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
</string>
   </property>
   <property name="text">
    <string>프로필</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>580</width>
     <height>320</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
